packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -20,6 +20,12 @@ let iconStrings: ICalendarIconStrings = {
   rightNavigation: rightArrow
 };
 
+let dateTimeFormatterCallbacks: ICalendarFormatDateCallbacks = {
+  formatMonthYear: (date: Date, strings: ICalendarStrings) => (strings.months[date.getMonth()] + ' ' + date.getFullYear()),
+  formatDay: (date: Date) => date.getDate().toString(),
+  formatYear: (date: Date) => date.getFullYear().toString()
+};
```

Comment:
Seems like we need a formatting function for the days of the week too

Reply:
See email for more details